sources/packages/web/Dockerfile

```diff
@@ -1,15 +1,21 @@
-FROM registry.access.redhat.com/ubi8/nodejs-16:1-5 as builder
+FROM registry.access.redhat.com/ubi8/nodejs-16:1-5 AS builder
 
 
 COPY ./package*.json ./
 RUN npm ci
 COPY . ./
 RUN npm run build
 
-FROM registry.access.redhat.com/ubi8/nodejs-16:1-5 as deployer
+FROM registry.access.redhat.com/ubi8/nginx-120 AS deployer
```

Comment:
Apparently, we also have version 1.22 available. Can we use it instead?
https://catalog.redhat.com/software/containers/ubi8/nginx-122/627245493b950f9f4eb01218